specs/wp-comments-spec.js

```diff
@@ -60,6 +60,8 @@ describe( `[${ host }] Comments: (${ screenSize })`, function() {
 		} );
 
 		step( 'Can post a reply', async function() {
+			// NOTE: we need to wait to prevent "You are posting comments too quickly. Slow down." error
+			await driver.sleep( 10000 );
```

Comment:
In which case this error happens and is 10s enough? Does `waitTillPresentAndDisplayed` can help instead of sleep?

Reply:
There is some sort of spam blocker which is rejecting comment submission it was right after the first one. Maybe there is a way to hack it, but since that delay is static - static timeout should do the job